tasks/release.yml

```diff
@@ -0,0 +1,14 @@
+version: '3'
+
+tasks:
+  snapshot:
+    desc: Creates a snapshot of the current working directory with files in .drainpipeignore excluded. The target directory should be provided as a cli argument e.g. task release:snapshot -- /tmp/release
+    cmds:
+      - if [ ! -f ".drainpipeignore" ]; then echo ".drainpipeignore does not exist" && exit 1; fi
+      - if [ -d "{{.CLI_ARGS}}" ] || [ "" == "{{.CLI_ARGS}}" ]; then echo "Please provide a path to a directory that does not yet exist" && exit 1; fi
+      - cp -R . {{.CLI_ARGS}}
+      - rm -rf {{.CLI_ARGS}}/.git
+      - cd {{.CLI_ARGS}} && git init
```

Comment:
Is the current directory preserved across multiple lines? I know with other CI tools each line would spawn a fresh shell, unless it was a single string block.

Reply:
(In particular, I wonder about the `rm` call below)

Reply:
It isn't, though you can preserve it with a pipe I believe